Update civicrm_group_roles.module

yurg opened the pull request:
Preventing Notice: Undefined index: #id in civicrm_group_roles_form_alter() (line 126 of sites\all\modules\civicrm\drupal\modules\civicrm_group_roles\civicrm_group_roles.module).

yurg committed:
Update civicrm_group_roles.module

Preventing Notice: Undefined index: #id in civicrm_group_roles_form_alter() (line 126 of sites\all\modules\civicrm\drupal\modules\civicrm_group_roles\civicrm_group_roles.module).

civicrm-builder commented:
Can one of the admins verify this patch?

jackrabbithanna commented:
Jenkins test this please

jackrabbithanna commented:
Jenkins test this please

seamuslee001 commented:
Jenkins ok to test

seamuslee001 commented:
Hi @yurg thanks for the PR are you able to fix the style issues found by our test bot https://test.civicrm.org/job/CiviCRM-Drupal-PR/104/checkstyleResult/new/

yurg committed:
Update civicrm_group_roles.module

Changes against https://test.civicrm.org/job/CiviCRM-Drupal-PR/104/checkstyleResult/new/

yurg committed:
Merge pull request #1 from yurg/yurg-patch-1

Update civicrm_group_roles.module

aiden-grau commented:
+1 on getting this merged...
